PatternPal/PatternPal.Tests/Utils/EntityNodeUtils.cs

```diff
@@ -231,6 +231,31 @@ private static SyntaxGraph CreateGraphFromInput(string INPUT)
             return graph;
         }
 
+        internal static SyntaxGraph CreateMethodWithParamaters()
+        {
+            const string INPUT = """
+                                 public class StringTest
+                                 {
+                                    internal StringTest StringTestFunction(IntTest testParam1, IntTest testParam2, StringTest testParam3)
+                                    {
+                                    }
+                                 }
+                                 public class IntTest
```

Comment:
This name is somewhat confusing when you refer to it. You refer to it as an Int node, which gives the impression the parameter has the type of int. But it is a class.